spec/factories/user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+
+	factory :user do
+    username 'bobiscool'
+    password 'legend101'
+    password_confirmation 'legend101'
+  end
```

Comment:
Lint/BlockAlignment: `end` at 7, 2 is not aligned with `factory :user do` at 3, 1